packages/fx-core/src/plugins/resource/apim/managers/apimManager.ts

```diff
@@ -110,8 +116,9 @@ export class ApimManager {
       apimConfig.serviceResourceId = getArmOutput(ctx, ApimArmOutput.ServiceResourceId);
       apimConfig.productResourceId = getArmOutput(ctx, ApimArmOutput.ProductResourceId);
       apimConfig.authServerResourceId = getArmOutput(ctx, ApimArmOutput.AuthServerResourceId);
-      apimConfig.publisherEmail = getArmOutput(ctx, ApimArmOutput.PublisherEmail);
-      apimConfig.publisherName = getArmOutput(ctx, ApimArmOutput.PublisherName);
+      // Remove publisherEmail and publisherName from config to avoid sensitive info being committed into git history
+      apimConfig.publisherEmail = undefined;
```

Comment:
can we just remove these 2 lines?

Reply:
No. Without setting these two properties to be undefined, they will be record into profile.xxx.json.